pom.xml

```diff
@@ -3,64 +3,90 @@
   <groupId>com.github.amnotbot</groupId>
   <artifactId>amnotbot-core</artifactId>
   <version>0.0.1-SNAPSHOT</version>
+
+  <scm>
```

Comment:
Ok. This is needed by the buildnumber-maven-plugin. 
I will update it to this github.